lib/jekyll/utils.rb

```diff
@@ -146,11 +134,13 @@ def parse_date(input, msg = "Input could not be parsed.")
     # Determines whether a given file has
     #
     # Returns true if the YAML front matter is present.
+    # rubocop: disable PredicateName
```

Comment:
We cannot rename public methods, no. Unless it was marked `private` before, it cannot be renamed.